webapp/channels/src/components/admin_console/session_length_settings.tsx

```diff
@@ -94,6 +110,7 @@ export default class SessionLengthSettings extends AdminSettings {
             sessionTimeoutSetting = (
                 <TextSetting
                     id='sessionIdleTimeoutInMinutes'
+                    type='number'
```

Comment:
here you are using it as `type='number'` and in other places as `type={'number'}` lets stick to one :)